### ISSUE TITLE
Lyrics tab got back to normal

### ISSUE THREAD

**Issue by XielQs** [sanoojes/Spicetify-Lucid#72]
### Do you have the latest versions of Spicetify, Spotify and Lucid?

- [X] I'm using the latest versions of Spicetify, Spotify and Lucid

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Is your problem definitely caused by the theme?

- [X] I have checked the app behavior without the theme and it is different

### Environment & Computer Info

```Markdown
- Spotify version: 1.2.48.405.gf2c48e6f
- Installed from: Arch User Repository (AUR)
- Spicetify version: 2.38.4
- Color scheme: dark
- Extensions:
- Custom apps:
- Lucid installation method: Spicetify Marketplace
```

### Description

Song lyrics got back to normal, can't see lyrics on black backgrounds

### Screenshots

![resim](https://github.com/user-attachments/assets/bd840483-f836-418e-996c-285ca3a0be36)

**Comment by sanoojes** [comment]
Yes, that was reverted to normal. I rewrote the entire theme from scratch and didn't have time to add those features yet.
